src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -11001,6 +11113,64 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/ActionBodyExecTemplate'
+  RuleSetPaging:
+    type: object
+    required:
+      - list
+    properties:
+      list:
+        type: object
+        required:
+          - pagination
+          - entries
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/RuleSetEntry'
+  RuleSetEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/RuleSet'
+  RuleSet:
+    type: object
+    required:
+      - id
+      - ownedBy
```

Comment:
This does sound like a more elegant solution. However there is one issue: should we still be doing this under the `linked-rule-set` entity then? If so that endpoint will do more than it claims, unless you consider that this operation is only done silently by the back end without the client ever knowing?